tests/test_gplates.py

```diff
@@ -0,0 +1,49 @@
+import pickle
+from gadopt import *
+from gadopt.gplates import GplatesFunction, pyGplatesConnector
+
+
+def test_gplates():
+    # Set up geometry:
+    rmin, rmax, ref_level, nlayers = 1.22, 2.22, 5, 16
+
+    # Construct a CubedSphere mesh and then extrude into a sphere
+    mesh2d = CubedSphereMesh(rmin, refinement_level=ref_level, degree=2)
+    mesh = ExtrudedMesh(
+        mesh2d,
+        layers=nlayers,
+        layer_height=(rmax - rmin)/(nlayers-1),
+        extrusion_type="radial",
+    )
+
+    V = VectorFunctionSpace(mesh, "CG", 2)
+
+    # compute surface velocities
+    rec_model = pyGplatesConnector(
+        rotation_filenames=[
+            '../demos/gplates_global/gplates_files/Zahirovic2022_CombinedRotations_fixed_crossovers.rot',
+        ],
+        topology_filenames=[
+            '../demos/gplates_global/gplates_files/Zahirovic2022_PlateBoundaries.gpmlz',
+            '../demos/gplates_global/gplates_files/Zahirovic2022_ActiveDeformation.gpmlz',
+            '../demos/gplates_global/gplates_files/Zahirovic2022_InactiveDeformation.gpmlz',
+        ],
```

Comment:
Maybe define `gplates_files = Path(__file__).parent / "demos/gplates_global/gplates_files"` and then pull the files out of there? Just to be sure with relative paths and pytest.

Reply:
Makes sense! Thanks Agnus.